client/me/help/help-contact/index.jsx

```diff
@@ -58,6 +59,9 @@ import {
 } from 'state/selectors';
 import QueryUserPurchases from 'components/data/query-user-purchases';
 import { getHelpSelectedSiteId } from 'state/help/selectors';
+import { getLanguage, isDefaultLocale } from 'lib/i18n-utils';
+const defaultLanguage = getLanguage( config( 'i18n_default_locale_slug' ) ).name;
```

Comment:
Can we move this const into the Module variables block?

Reply:
Sure thing, good catch.